lib/oidcc/token/access.ex

```diff
@@ -33,17 +33,26 @@ defmodule Oidcc.Token.Access do
           endpoint :: String.t(),
           client_context :: ClientContext.t()
         ) :: %{String.t() => String.t()}
+  @spec authorization_headers(
+          access_token :: t(),
+          method :: :get | :post,
+          endpoint :: String.t(),
+          client_context :: ClientContext.t(),
+          opts :: %{optional(:dpop_nonce) => binary()}
```

Comment:
Can you define the `opts` as `oidcc_token:authorization_headers_opts/0`?

Reply:
(Same for https://github.com/erlef/oidcc/blob/d57635e8a7bea3352ab2b4091fc1ec679663f884/src/oidcc_token.erl#L957)